app/views/posts/index.html.erb

```diff
@@ -1,25 +1,36 @@
 <h1>User All Posts</h1>
-<h2>User 1</h2>
-<div>
-  <h3>Post 1</h3>
-  <p>Amet dictum sit amet justo donec enim diam vulputate ut. Etiam sit 
-    amet nisl purus in mollis nunc. Aenean euismod elementum nisi quis 
-    eleifend quam adipiscing vitae. Habitasse platea dictumst quisque 
-    sagittis purus.</p>
-  <ul>
-    <li>User 3 - Comment 1</li>
-    <li>User 4 - Comment 2</li>
-    <li>User 5 - Comment 3</li>
-  </ul>
-</div>
-<div>
-  <h3>Post 2</h3>
-  <p>Duis convallis convallis tellus id interdum velit laoreet id. 
-    Diam sit amet nisl suscipit adipiscing bibendum est ultricies. 
-    Eu facilisis sed odio morbi quis commodo odio aenean.</p>
-  <ul>
-    <li>User 6 - Comment 1</li>
-    <li>User 4 - Comment 2</li>
-    <li>User 5 - Comment 3</li>
-  </ul>
+<div class="user-line">
+    <%= link_to image_tag(@user.photo, class: 'user-photo'), user_path(@user) %>
+    <div class="name-counter">
+      <span class="name">
+        <%= link_to @user.name, user_path(@user) %>
+      </span>
+      <div>
+        <span class="right-top"></span>
+        <span class="counter">Number of posts: <strong><%= @user.posts_counter %></strong></span>
+        <span class="right"></span>
+      </div>
+    </div>
 </div>
+
+<% @posts.each do |post| %>
+  <div class="post-line">
+    <h3>Post <%= post.id %>: <%= post.title %>
+      <span class="right1"><%= link_to "Read More", post_path(post) %></span>
```

Comment:
- [ ] 🌟 Good job on the implementation! However, there's a slight adjustment needed for the redirect link. As per the wireframe, the link should redirect to the `users/1/posts/1` route, but it's currently redirecting to `posts/1`. To align with the wireframe specifications, I recommend updating your code to ensure the correct redirect path (`users/1/posts/1`).

| Your | Design |
| --- | --- |
|  ![image](https://github.com/jcmunav63/my-ror-blog/assets/55840999/76ad2bce-0741-47c6-9464-42a8409c33a6)| ![image](https://github.com/jcmunav63/my-ror-blog/assets/55840999/3cf37abf-c9b1-4669-9359-5424dabecf4b)|

Keep up the great work, and if you have any questions or need further guidance, feel free to reach out! 🚀👩‍💻